docs/pages/ui-programming/react-native-toast.md

```diff
@@ -0,0 +1,117 @@
+---
+title: How to display a toast in React Native
+sidebar_title: Displaying toast messages
+---
+
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+
+## What is a toast?
+
+Toasts are the standard technique in mobile development for notifying your users about something without interrupting what they are doing.
+
+According to the [Android Developers
+Documentation](https://developer.android.com/guide/topics/ui/notifiers/toasts): "A toast provides
+simple feedback about an operation in a small popup. It only fills the amount of space required for
+the message and the current activity remains visible and interactive. Toasts automatically disappear
+after a timeout".
+
+To present a toast, we recommend two solutions: an API from React Native itself and a library
```

Comment:
```suggestion
To present a toast, we recommend two solutions: an API from the `react-native` package and a library
```
this wording communicates better that it's not really a core component and more an arbitrary component